queue_job_cron/models/ir_cron.py

```diff
@@ -32,14 +32,17 @@ def _run_job_as_queue_job(self, server_action):
         return server_action.run()
 
     def method_direct_trigger(self):
-        if self.run_as_queue_job:
-            return self.with_delay(
-                priority=self.priority,
-                description=self.name,
-                channel=self.channel_id.complete_name,
-            )._run_job_as_queue_job(server_action=self.ir_actions_server_id)
-        else:
-            return super().method_direct_trigger()
+        for cron in self:
+            if not cron.run_as_queue_job:
+                super().method_direct_trigger()
```

Comment:
the call to super here should be done using cron instead of self

Reply:
You mean like this? 
```
super(IrCron, cron).method_direct_trigger()
``` 
Funny. You proposed the opposite thing yesterday and I was thinking about this to short and didn’t remember why I did it like this, so just took your suggestion. ;-)

Reply:
Thx anyway.

Reply:
eh, sorry, I've deleted the comment right after as I realized it was correct :stuck_out_tongue_winking_eye: 
Maybe you still had it in your notification :man_shrugging: